admin/settings-about-mesh.php

```diff
@@ -7,132 +7,188 @@
+								<div class="about-box-icon">
+									<img src="<?php echo esc_url( LINCHPIN_MESH___PLUGIN_URL . 'assets/images/feature-responsive.svg' ); ?>"/>
+								</div>
+								<div class="about-box-copy">
+									<h4 class="no-margin"><?php esc_html_e( 'Fully Responsive', 'mesh' ); ?></h4>
+									<p><?php esc_html_e( 'Rest easy knowing your content will display seamlessly across browsers and devices.', 'mesh' ); ?></p>
+								</div>
+							</div>
+						</div>
 
-                        <div class="mesh-columns-6" >
-                            <div class="grey-box" data-equalizer-watch="">
-                                <div class="about-box-icon">
-                                    <img src="<?php echo ( LINCHPIN_MESH___PLUGIN_URL . 'assets/images/feature-plays-well.svg'); ?>" />
-                                </div>
-                                <div class="about-box-copy">
-                                    <h4 class="no-margin"><?php esc_html_e( 'Plays Well with Others', 'mesh' ); ?></h4>
-                                    <p>
-										<?php printf( __( 'Continually updated with hooks and filters to extend functionality. For a full list, check out <a href="%s" target="_blank" rel="noopener">MeshPlugin.com</a>.', 'mesh'),
-											esc_url( 'https://meshplugin.com/knowledge-base' ) ); ?>
+						<div class="mesh-columns-6">
+							<div class="grey-box" data-equalizer-watch="">
+								<div class="about-box-icon">
+									<img src="<?php echo esc_url( LINCHPIN_MESH___PLUGIN_URL . 'assets/images/feature-plays-well.svg' ); ?>"/>
+								</div>
+								<div class="about-box-copy">
+									<h4 class="no-margin"><?php esc_html_e( 'Plays Well with Others', 'mesh' ); ?></h4>
+									<p>
+										<?php
+											printf(
+												// translators: %s Mesh Knowlegebase URL.
+												wp_kses( __( 'Continually updated with hooks and filters to extend functionality. For a full list, check out <a href="%s" target="_blank" rel="noopener">MeshPlugin.com</a>.', 'mesh' ), $safe_content ),
+												esc_url( 'https://meshplugin.com/knowledge-base' )
+											);
+										?>
 									</p>
-                                </div>
-                            </div>
-                        </div>
-                    </div>
-                </div>
-            </div>
+								</div>
+							</div>
+						</div>
+					</div>
+				</div>
+			</div>
 
-            <div class="wrapper mesh-row">
-                <h2 class="color-darkpurple light-weight"><?php esc_html_e('About the Team Behind Mesh', 'mesh' ); ?></h2>
-                <div class="about-devs-container mesh-columns-12">
-                    <div class="no-margin">
-                        <p class="no-margin">
-                            <?php
-                            printf( __( '<a href="%s">Linchpin</a> is a Digital Agency that specializes in WordPress. 
+			<div class="wrapper mesh-row">
```

Comment:
We should probably `target="_blank"` the links below for Linchpin, Jetpack Pro, etc.